flink-rpc/flink-rpc-akka/src/test/java/org/apache/flink/runtime/rpc/akka/TimeoutCallStackTest.java

```diff
@@ -77,9 +79,19 @@ void stopTestEndpoints() {
 
     @Test
     void testTimeoutException() throws Exception {
```

Comment:
```suggestion
    void testTimeoutExceptionWithTime() throws Exception {
```
nit: Can we make this test as descriptive as the other one?